acstools/acs_destripe_plus.py

```diff
@@ -255,6 +255,14 @@ def destripe_plus(inputfile, suffix='strp', stat='pmode1', maxiter=15,
     cte_correct : bool
         Perform CTE correction.
 
+    keep_intermediate_files : bool
```

Comment:
LoL @jryon , looks like it was already implemented here!

Reply:
Haha, nice! Is this PR going to be merged eventually? I can close my PR if so.

Reply:
Yes, that's what I meant.

Reply:
OK nvm. This PR survives...